ray-operator/test/e2e/raycluster_test.go

```diff
@@ -0,0 +1,70 @@
+package e2e
+
+import (
+	"testing"
+	"time"
+
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	rayv1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1"
+	"github.com/ray-project/kuberay/ray-operator/controllers/ray/utils"
+	rayv1ac "github.com/ray-project/kuberay/ray-operator/pkg/client/applyconfiguration/ray/v1"
+	. "github.com/ray-project/kuberay/ray-operator/test/support"
+)
+
+func TestRayClusterManagedBy(t *testing.T) {
```

Comment:
I think it would be good to also test immutability as we do for RayJob: https://github.com/ray-project/kuberay/pull/2589/files#diff-1c4af89d9d063e6f2d26ec46badd713b89a17b92745ccb564c44e13ee2d33be0R269-R278

Reply:
done